src/raft.rs

```diff
@@ -938,7 +934,7 @@ impl<T: Storage> Raft<T> {
 
     fn num_pending_conf(&self, ents: &[Entry]) -> usize {
         ents.iter()
-            .filter(|e| e.get_entry_type() == EntryType::EntryConfChange)
+            .filter(|e| e.entry_type() == EntryType::EntryConfChange)
```

Comment:
Not possible to remove the getters on these too? Or later work?

Reply:
Nope. The field is `i32` while we want to get an enum.

This getter is actually generated by prost by default.